_config.yml

```diff
@@ -37,8 +37,10 @@ enterpriseplan: Tiny Custom Plans
 supporturl: https://support.tiny.cloud
 plugindirectory: https://www.tiny.cloud/tinymce/features/
 gettiny: https://www.tiny.cloud/get-tiny
+blogurl: https://www.tiny.cloud/blog
 download-community: https://www.tiny.cloud/get-tiny/self-hosted/
 download-enterprise: https://www.tiny.cloud/my-account/downloads/
+download-zip: https://download.tiny.cloud/tinymce/community/tinymce_latest.zip
```

Comment:
Should this be `download-community-zip`? As that'd be clearer and we may need an enterprise zip at some stage potentially :shrug: (though not overly likely atm)

Reply:
Removed. after a quick search, I can't find anywhere that I've used it :thinking: I may have added it then changed the procedure?

Reply:
fixed in https://github.com/tinymce/tinymce-docs/pull/2196/commits/9eac4d2b0b024228dc88cacbb7ef58c295ac5477